### PR TITLE
updates based on user feedback

### PR DESCRIPTION
**Pull request by BillWagner**
Users had two issues with the transition from dot.net to docs.

1. The first step of the list quick start is repeated. I added a note to explain that.
1. Several people already had an editor, and the instructions to install visual studio set them back. I added a not to make it more clear that you only need this if you don't already have an editor.

/cc @rowanmiller

### CODE REVIEW

File: docs/csharp/quick-starts/arrays-and-collections.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ class.
 > [!NOTE]
 > If you're starting from the code you wrote in [dot.net](https://dot.net/), you already have the code written in this section. Jump to [Modify list contents](#modify-list-contents).
 
-This lesson assumes you've finished the online quick starts, and you've installed the [.NET Core SDK](http://dot.net/core) and [Visual Studio Code](https://code.visualstudio.com/). 
+This lesson assumes you've finished the online quick starts, and you've installed the [.NET Core SDK](http://dot.net/core) and that you have a code editor. If you don't have one, we recommend [Visual Studio](https://www.visualstudio.com/) for Mac or Windows.
```
Comment (rpetrusha) [inline docs/csharp/quick-starts/arrays-and-collections.md]: And is overused here. How about: "This lesson assumes that you've finished the online quick starts, you've installed..., and you have a code editor..."

**Review by rpetrusha — APPROVED** [review]
One suggested change, @BillWagner, but otherwise LGTM.

File: docs/csharp/quick-starts/arrays-and-collections.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ class.
 > [!NOTE]
 > If you're starting from the code you wrote in [dot.net](https://dot.net/), you already have the code written in this section. Jump to [Modify list contents](#modify-list-contents).
 
-This lesson assumes you've finished the online quick starts, and you've installed the [.NET Core SDK](http://dot.net/core) and [Visual Studio Code](https://code.visualstudio.com/). 
+This lesson assumes you've finished the online quick starts and you've installed the [.NET Core SDK](http://dot.net/core). You also need a code editor. If you don't have one, we recommend [Visual Studio](https://www.visualstudio.com/) for Mac or Windows.
```
Comment (rowanmiller) [inline docs/csharp/quick-starts/arrays-and-collections.md]: Yep, this fixes the confusion 😄

Comment (rowanmiller) [reply docs/csharp/quick-starts/arrays-and-collections.md]: I wonder if we're making it sound like there is no option for Linux? Visualstudio.com is pretty responsive to the OS you are on, so I wonder if we could just say that Visual Studio has offerings for Windows, Mac, and Linux.

Comment (BillWagner) [reply docs/csharp/quick-starts/arrays-and-collections.md]: Should I call out Visual Studio Code on Linux? I thought of that, but thought it might be too wordy. Thoughts?

File: docs/csharp/quick-starts/list-collection.yml [hunk 3]
```diff
@@ -17,6 +17,10 @@ items:
   content: |
     Run the following code in the interactive window. To do that, type the following code block in the interactive window (replace "<name>" with your name) and click the **Run** button:
 
+    > [!NOTE]
+    > If you're starting from the code you wrote in [dot.net](https://dot.net/), this step is repeated. Copy and paste the code, and then move on to the next steps.
+
```
Comment (rowanmiller) [inline docs/csharp/quick-starts/list-collection.yml]: Maybe this is better? dot.net is just a redirect, so we should point to the site URL. There will be a more specific URL once the new content is published.

> If you're coming from the [.NET In-Browser Tutorial](https://microsoft.com/net), you've already completed this step. Copy and paste the following code into the interactive window, then move on to [the next step](<give them an easy link>).

File: docs/csharp/quick-starts/arrays-and-collections.md (hunk 2 above)
Comment (BillWagner) [reply docs/csharp/quick-starts/arrays-and-collections.md]: I just made an update here. Let me know what you think of the wording.

Comment (rowanmiller) [reply docs/csharp/quick-starts/arrays-and-collections.md]: Yep, I think that does the trick

**Review by rowanmiller — APPROVED** [review]
